Fabric.Platform.Auth/AuthorizationMiddleware.cs

```diff
@@ -33,5 +34,28 @@ public static AppFunc Inject(AppFunc next, string[] requiredScopes, string[] all
                 return Task.CompletedTask;
             };
         }
+
+        private static bool HasPath(string[] allowedPaths, string pathToCheck)
+        {
+            foreach(var path in allowedPaths)
+            {
+                if(path.Last() == '*')
+                {
+                    if(pathToCheck.StartsWith(path.TrimEnd('*')))
+                    {
+                        return true;
+                    }
+                }
+                else
+                {
```

Comment:
Do we want case this to be case-insensitive?

Reply:
Before, the logic used Contains, which is case-sensitive.  I dont think there is a need for making it less restrictive.  Im going to skip it for now, but if there is a good reason we can always change it later.